### ISSUE TITLE
outline only auto-updating on errors, but not on "all fine"

### ISSUE THREAD

**Issue by GitMensch** [redhat-developer/vscode-xml#359]
setup: xml file with an xmlns entry that is matched by xml catalog - and validates without errors.
opened the catalog file in another tab, break the matching entry, save.

result: problems pane show the error, when switching to the xml file the outline shows the error markers on the appropriate places which is perfect.

Then switch back and unbreak the catalog entry, save.
result: problem pane is empty

issue: when now switching to the xml file the outline still contain the error markers.
workaround: add a whitespace and remove it, outline is updated (only reported because it gets auto-updated with the error, just not when the error vanishes

**Comment by angelozerr** [comment]
@GitMensch you are right,I never noticed this bug,  but we don't manage that, it's vscode with LSP support which manages that. We should report this problem on vscode side.

**Comment by GitMensch** [comment]
Please do so as I have nearly zero knowledge on this topic, just recognized the issue.
